CMakeLists.txt.in

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 3.5)
+project(build-seastar NONE)
+
+include(ExternalProject)
+
+ExternalProject_Add(seastar_ext
+  GIT_REPOSITORY    https://github.com/scylladb/seastar.git
+  GIT_TAG           9ed5fca9207f81662dd205b5c778890165959f3f
+  CONFIGURE_COMMAND  ./configure.py --mode=release --enable-dpdk --without-tests --without-apps --without-demos
+  BUILD_IN_SOURCE    1
+  BUILD_COMMAND      ninja -C build/release
+  INSTALL_COMMAND    "")
+
+ExternalProject_Add(zstd_ext
```

Comment:
we should port over all the other recipes in the cooking_recipe.cmake from seastar so we manage them here tho and we don't have to call cooking at all. what do you think ?

Reply:
are you referring to cooking.sh in smf, or cooking that's ran by the seastar build system? for the former, yeh--definitely will want to completely remove cooking from smf. for seastar, same, but less urgent. is there something that'd push that into v1 vs v2?